content/workshops/_index.md

```diff
@@ -5,8 +5,10 @@ date: 2016-04-09 16:50:16+02:00
 pre: <b>3. </b>
 title: Workshops
 weight: 3
+ready: true
 ---
 
+
```

Comment:
We dont do workshops. So this isn't actually ready.

It's important to think about what ready means. This task is not to just update frontmatter without thinking about what it means